build/cloud_builder.go

```diff
@@ -62,8 +62,7 @@ func (c *CloudBuilder) GetBuildPipeLine() ([]func() error, error) {
 	if c.local.IsOnlyCopy() {
 		buildPipeline = append(buildPipeline,
 			c.local.ExecBuild,
-			c.local.UpdateImageMetadata,
-			c.local.PushToRegistry)
+			c.local.UpdateImageMetadata)
```

Comment:
will it affect cloud build? I guess cloud build should push image

Reply:
nop,only if CMD or RUN commands exist in kubefile will push image , if only copy ,equal local build.